guide/src/class.md

```diff
@@ -1150,14 +1155,14 @@ PyO3 adds a class attribute for each variant, which may be used to construct val
 enum Shape {
```

Comment:
This enum was the same in both instances, while now the first example has Square instead of RegularPolygon. I think it's best to keep it the same even with the tuple variant example.